droid-hal-device.inc

```diff
@@ -326,7 +326,12 @@ shopt -u nullglob
 %{dhd_path}/helpers/makefstab --files $FSTAB_FILES  --skip auto /acct /boot /cache /data /misc /recovery /staging /storage/usbdisk /sys/fs/cgroup /sys/fs/cgroup/memory /sys/kernel/debug  /sys/kernel/config /dev/usb-ffs/adb %{?makefstab_skip_entries} --outputdir tmp/units
 
 echo Fixing up mount points
+if [ -f rpm/fixup-mountpoints ];
+then
+rpm/fixup-mountpoints tmp/units/*
```

Comment:
Unfortunately this can't be done in this way, because fixup-mountpoints is used by hybris-boot itself [see here](https://github.com/mer-hybris/hybris-boot/blob/dd07cd4cd32682f079c7756fc369d23ca82139f7/Android.mk)